fpage/models.py

```diff
@@ -24,10 +25,10 @@ def __str__ (self):
         return self.song_title 
 
 class UserProfile(models.Model):
-    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    user = models.OneToOneField(User,on_delete=models.CASCADE)
     first_name=models.CharField(max_length=50)
     last_name=models.CharField(max_length=50)
     email=models.CharField(max_length=150)
-    phone_num=models.IntegerField()
+    phone_num=PhoneNumberField(null=False, blank=False, unique=True)
```

Comment:
Make the phone number field both `null=True` and `blank=True`. Phone number should be a optional field.

Reply:
ok